### ISSUE TITLE
NTR: MRI: tectal enlargement (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9006]
New term request MRI: tectal enlargement (elthat_160819023926): Tectum  UMLS: 2748980

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000284
- label: Tectal plate enlargement
- parents: [Abnormal brainstem morphology (HP:0002363)](https://hpo.jax.org/app/browse/term/HP:0002363)
- pmid: 
- def: Abnormal increase in size of the tectal plate (also known as the tectum or the quadrigeminal plate). The tectum is the dorsal portion of the midbrain (brainstem).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9006
- synonyms: Quadrigeminal plate enlargement(NONE)|Tectal enlargement(NONE)
